configs/mot/qdtrack/qdtrack_faster-rcnn_r50_fpn_4e_dancetrack.py

```diff
@@ -0,0 +1,130 @@
+_base_ = [
+    '../../_base_/models/faster_rcnn_r50_fpn.py',
+    '../../_base_/datasets/dancetrack.py', '../../_base_/default_runtime.py'
+]
```

Comment:
Use base config of https://github.com/open-mmlab/mmtracking/blob/master/configs/mot/qdtrack/qdtrack_faster-rcnn_r50_fpn_4e_mot17-private-half.py